src/AssetPackages.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+
+namespace WonderWp\Component\Asset;
+
+
+use Symfony\Component\Asset\Exception\InvalidArgumentException;
+use Symfony\Component\Asset\Exception\LogicException;
+use Symfony\Component\Asset\PackageInterface;
+use Symfony\Component\Asset\Packages;
+use WonderWp\Component\Asset\Exception\PackageReservedNameException;
+
+class AssetPackages
+{
+    /**
+     * @var PackageInterface|null
+     */
+    private $defaultPackage;
+    private $packages = [];
+
+    public function __construct(PackageInterface $defaultPackage = null, iterable $packages = [])
+    {
+        $this->defaultPackage = $defaultPackage;
+
+        foreach ($packages as $name => $package) {
+            if ($name === 'default') {
+                throw new PackageReservedNameException('The package name \'default\' is reserved for default package');
+            }
+
+            $this->addPackage($name, $package);
+        }
+    }
+
+    public function setDefaultPackage(PackageInterface $defaultPackage)
+    {
+        $this->defaultPackage = $defaultPackage;
+    }
+
+    public function addPackage(string $name, PackageInterface $package)
+    {
+        $this->packages[$name] = $package;
+    }
+
+    public function getPackages(): array
+    {
+        if ($this->defaultPackage !== null) {
+            return ['default' => $this->defaultPackage] + $this->packages;
+        }
+
+        return $this->packages;
+    }
+
+    /**
+     * Returns an asset package.
+     *
+     * @param string $name The name of the package or null for the default package
+     *
+     * @return PackageInterface An asset package
+     *
+     * @throws InvalidArgumentException If there is no package by that name
+     * @throws LogicException           If no default package is defined
+     */
+    public function getPackage(string $name = null)
+    {
+        if (null === $name) {
```

Comment:
Il existe une fonction is_null en php je crois.
```suggestion
        if (is_null($name)) {
```
idem pour la ligne suivante.